cmd/peggo/orchestrator.go

```diff
@@ -192,6 +196,16 @@ func getOrchestratorCmd() *cobra.Command {
 				return err
 			}
 
+			providers := konfig.Strings(flagOracleProviders)
+			newOracle, err := oracle.New(context.Background(), logger, providers)
```

Comment:
Let's think of a better var than `newOracle`, even `o` would suffice I think :)

Reply:
done at https://github.com/umee-network/peggo/pull/231/commits/f25d8f891993cb275a8065040111dd9ee91c66ff